softlayer/resource_softlayer_file_storage.go

```diff
@@ -723,16 +723,24 @@ func updateAllowedIpAddresses(d *schema.ResourceData, sess *session.Session, sto
 			if len(ipObject) != 1 {
 				return fmt.Errorf("Number of IP address is %d", len(ipObject))
 			}
-			_, err = services.GetNetworkStorageService(sess).
-				Id(id).
-				AllowAccessFromHostList([]datatypes.Container_Network_Storage_Host{
-					{
-						Id:         ipObject[0].Id,
-						ObjectType: sl.String("SoftLayer_Network_Subnet_IpAddress"),
-					},
-				})
-			if err != nil {
-				return err
+
+			for {
+				_, err = services.GetNetworkStorageService(sess).
+					Id(id).
+					AllowAccessFromHostList([]datatypes.Container_Network_Storage_Host{
+						{
+							Id:         ipObject[0].Id,
+							ObjectType: sl.String("SoftLayer_Network_Subnet_IpAddress"),
+						},
+					})
+				if err != nil {
+					if strings.Contains(err.Error(), "SoftLayer_Exception_Network_Storage_Group_MassAccessControlModification") {
+						time.Sleep(5 * time.Second)
```

Comment:
This looks good. Let's make the time we wait between retries a constant `RetryTime` that can be used in all the relevant places.